### DISCUSSION TITLE
Newbie here. Q re upgrading Flameshot

### DISCUSSION THREAD

**Discussion by anandananda** [flameshot-org/flameshot#2720] [Q&A] [answered]
I'm using Flameshot 11 (deb package) on my Ubuntu 22.04 Gnome/Wayland system. It's working good. It's telling me that ver 12 is available. I click on the 'Software Updater' (SU) icon tool everyday to update my system. Will Flameshot 12 be automatically included in a future SU update? Or do I need to update Flameshot manually?

And how can I tell if a deb package app will (or will not) be automatically updated by SU?

Thanks!

**Comment by mmahmoudian** [comment, marked as answer]
That is something that you should ask Ubuntu people, especially the maintainer of Flameshot in Ubuntu repository. If you want to stay up to date regardless of Ubuntu people's decision, you ca go with a distro-agnostic solution like Flatpak, Snap, or AppImage. For more information read the following:

https://flameshot.org/docs/installation/installation-linux/#distro-agnostic
